src/main/java/com/xiaomi/infra/pegasus/operator/client_operator.java

```diff
@@ -5,33 +5,39 @@
 
 import com.xiaomi.infra.pegasus.base.error_code;
 import com.xiaomi.infra.pegasus.base.gpid;
+import com.xiaomi.infra.pegasus.replication.request_meta;
 import com.xiaomi.infra.pegasus.rpc.ThriftHeader;
 import com.xiaomi.infra.pegasus.thrift.TException;
-import com.xiaomi.infra.pegasus.tools.Tools;
 
 public abstract class client_operator {
   public client_operator(gpid gpid, String tableName) {
     this.header = new ThriftHeader();
-    this.header.app_id = gpid.get_app_id();
-    this.header.partition_index = gpid.get_pidx();
+    this.meta = new request_meta();
+    this.meta.setApp_id(gpid.get_app_id());
+    this.meta.setPartition_index(gpid.get_pidx());
```

Comment:
`setApp_id` and `setPartition_index` is so strange in java, suggest rename `setAppId` and `setPartitionIndex`